### PR TITLE
token authentication (backend) + displaying my chores on chores page …

### PR DESCRIPTION
**Pull request by swarakurakula**
…(from db)

### CODE REVIEW

File: packages/express-backend/backend.js
Comment (AlexiLynn) [inline packages/express-backend/backend.js:1]: Great work on the authentication. I appreciate your comment as well.

File: packages/express-backend/services.js
Comment (AlexiLynn) [inline packages/express-backend/services.js:1]: Thank you for blazing a trail on connecting frontend with backend. Perhaps the function names could be shorter for simplicity?

File: packages/frontend/src/components/Chores/ChoresLayout.js
Comment (AlexiLynn) [inline packages/frontend/src/components/Chores/ChoresLayout.js:1]: I appreciate your detailed comments. It makes it easier to read through. Thank you.

File: packages/frontend/src/components/Layout.js
Comment (AlexiLynn) [inline packages/frontend/src/components/Layout.js:1]: Thank you for bringing this to my attention. I will make sure to add this next commit.

File: packages/frontend/src/pages/LogIn.js
Comment (AlexiLynn) [inline packages/frontend/src/pages/LogIn.js:1]: Yay! I am so glad this is finally implemented.

**Review by AlexiLynn — COMMENTED** [review]
Great Job!

**Review by Sombaudy — APPROVED** [review]

**Comment by jcolt45** [comment]
Hi Swara. The backend query endpoints and services look good. I created a test user in the database with the signup endpoint and was able to authenticate with the signin endpoint. I noticed that the user is created a "__v" variable, I assume its part of the authentication, but if you could clarify that would be great. Great work!

**Comment by Sombaudy** [comment]
No comments. All good to me.

**Comment by swarakurakula** [comment]
> Hi Swara. The backend query endpoints and services look good. I created a test user in the database with the signup endpoint and was able to authenticate with the signin endpoint. I noticed that the user is created a "__v" variable, I assume its part of the authentication, but if you could clarify that would be great. Great work!

response: from my understanding its something called VersionKey and it is automatically created by default
the definition below is taken from: https://mongoosejs.com/docs/guide.html#versionKey
"The versionKey is a property set on each document when first created by Mongoose. This keys value contains the internal [revision](http://aaronheckmann.blogspot.com/2012/06/mongoose-v3-part-1-versioning.html) of the document. The versionKey option is a string that represents the path to use for versioning. The default is __v."

**Pull request merged**